users/apps.py

```
@@ -4,3 +4,6 @@
 class UsersConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'users'
+
+    def ready(self):
+        import users.signals
```

Ruff (0.7.0)

`users.signals` imported but unused

Remove unused import: `users.signals`

(F401)